packages/ui-components/src/components/text-field/text-field.types.ts

```diff
@@ -66,4 +66,8 @@ export interface ITextField {
 	forcedFocus?: boolean;
 	/** (optional) Text field tooltip configuration */
 	tooltipConfig?: Partial<ITooltip>;
+	/** (optional) Use a input mask when the text field type is number (default true) */
+	useInputMask?: boolean;
+	/** (optional) Input mask regex */
+	inputMaskRegex?: string;
```

Comment:
should we use `RegExp` type for this? (we would need to check if `Inputmask` accepts it or not, otherwise it may not have any benefit)